single_tech_samples/synapseanalytics/sample3_integration_testing/README.md

```diff
@@ -0,0 +1,180 @@
+# DataOps - Synapse Integration Testing <!-- omit in toc -->
+
+This sample showcases additional integration tests for Azure Synapse. It includes the deployment of infrastructure based on the E2E sample [DataOps - Parking Sensor (Synapse)](https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/main/e2e_samples/parking_sensors_synapse).
+
+## Contents <!-- omit in toc -->
+
+- [Solution Overview](#solution-overview)
+  - [Technologies used](#technologies-used)
+- [Key Concepts](#key-concepts)
+  - [Environments](#environments)
+  - [Testing](#testing)
+- [How to use the sample](#how-to-use-the-sample)
+  - [Prerequisites](#prerequisites)
+    - [Software pre-requisites if you use dev container](#software-pre-requisites-if-you-use-dev-container)
+  - [Setup and Deployment](#setup-and-deployment)
```

Comment:
As I was walking through this README, it isn't clear to the reader that in order to use DevContainer, they have to copy the .envtemplate, and create a .env file. Optionally then having the ability to populate it or leave it blank.

![Uploading image.png…]()

Reply:
I see the call out, I think it may be worth moving lines 113 - 123 above the section that guides the user on using Az CLI to set their session.

This is so that if individuals want to use a devcontainer, they do that as the very first thing, rather than rely on them installing Az CLI manually to follow the instructions, then realizing they didn't have to do that since the devcontainer installs AZ CLI for them

Reply:
Updated the README